bleep-cli/src/scala/bleep/commands/InstallZshTabCompletions.scala

```diff
@@ -32,7 +32,7 @@ case class InstallZshTabCompletions(userPaths: UserPaths, logger: Logger, stdout
       println(completionScript)
     } else {
       logger.info(s"Writing $completionScriptDest")
-      FileSync.softWriteBytes(completionScriptDest, completionScript.getBytes(StandardCharsets.UTF_8)): Unit
+      FileSync.softWriteBytes(completionScriptDest, completionScript.getBytes(StandardCharsets.UTF_8)).discard()
```

Comment:
Why not just
```scala
val _ = FileSync.softWriteBytes(completionScriptDest, completionScript.getBytes(StandardCharsets.UTF_8))
```
There's no need to invent custom extension methods to do a simple thing like this...

Reply:
Believe it or not, cross compiling between strict 2.13 and 3 provides no way to resolve all unused cases. And you need to use 3-4-5 different syntaxes to resolve most unused cases. It's laughable, really. Inventing this is way easier to deal with

Reply:
Oh boy :scream_cat: fair enough then...